pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -154,37 +154,42 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, logge
 
 // TODO: Currently we create a single dummy prowjob just for testing. The actual implementation
 // will be introduced in https://issues.redhat.com/browse/DPTP-2577
-func generateProwjobs(org, repo, branch, prpqrName, prpqrNamespace string) []prowv1.ProwJob {
-	labels := map[string]string{
-		v1.PullRequestPayloadQualificationRunLabel: prpqrName,
-	}
+func generateProwjobs(org, repo, branch, prpqrName, prpqrNamespace string, releaseJobSpec []v1.ReleaseJobSpec) []prowv1.ProwJob {
+	var ret []prowv1.ProwJob
+	for _, releaseJob := range releaseJobSpec {
+		labels := map[string]string{
+			v1.PullRequestPayloadQualificationRunLabel: prpqrName,
+		}
 
-	base := prowconfig.JobBase{
-		Agent: string(prowv1.KubernetesAgent),
-		Name:  fmt.Sprintf("dummy-pj-for-%s", prpqrName),
-		Spec: &corev1.PodSpec{
-			Containers: []corev1.Container{{Image: "centos:8", Command: []string{"sleep"}, Args: []string{"100"}}},
-		},
+		base := prowconfig.JobBase{
+			Agent: string(prowv1.KubernetesAgent),
+			Name:  fmt.Sprintf("dummy-%s-for-%s", releaseJob.Test, prpqrName),
+			Spec: &corev1.PodSpec{
+				Containers: []corev1.Container{{Image: "centos:8", Command: []string{"sleep"}, Args: []string{"100"}}},
+			},
 
-		UtilityConfig: prowconfig.UtilityConfig{
-			Decorate: utilpointer.BoolPtr(true),
-		},
-	}
+			UtilityConfig: prowconfig.UtilityConfig{
+				Decorate: utilpointer.BoolPtr(true),
+			},
+		}
 
-	extraRefs := prowv1.Refs{
-		Org:     org,
-		Repo:    repo,
-		BaseRef: branch,
-	}
-	base.ExtraRefs = []prowv1.Refs{extraRefs}
+		extraRefs := prowv1.Refs{
+			Org:     org,
+			Repo:    repo,
+			BaseRef: branch,
+		}
+		base.ExtraRefs = []prowv1.Refs{extraRefs}
 
-	periodicJob := prowconfig.Periodic{
-		JobBase: base,
-		Cron:    "@yearly",
-	}
+		periodicJob := prowconfig.Periodic{
+			JobBase: base,
+			Cron:    "@yearly",
+		}
+
+		pj := pjutil.NewProwJob(pjutil.PeriodicSpec(periodicJob), labels, nil)
+		pj.Name = fmt.Sprintf("dummy-pj-for-%s", prpqrName)
```

Comment:
This does not look right - this could result in creating multiple ProwJobs with the same name. I think using the UUIDs (just like for normal ProwJobs is enough here) should be fine here.